docker/mint/run.sh

```diff
@@ -11,14 +11,16 @@ set -ex
 IMAGE="${1:-libra_mint:latest}"
 AC_HOST="${2:-172.18.0.10}"
 AC_PORT="${3:-8080}"
-LOG_LEVEL="${4:-info}"
+CHAIN_ID="${4:-TESTING}"
```

Comment:
so this will be different for devnet and testnet, fwiu this script is used for both.
for devnet, should be "DEVNET" or 3, for testnet, should be "TESTNET" or 2

Reply:
Didn't see #5098. I will close this one.  Let's move the conversation there.